MainProgram/DataStructuresTests/OpenAddressingHashTableTest.cs

```diff
@@ -0,0 +1,235 @@
+﻿using System;
```

Comment:
All new unit tests should be added to the `UnitTest` project. The `MainProgram` project is deprecated and will be removed soon. Please move this to the `UnitTest` project.